README.md

```diff
@@ -116,4 +116,4 @@ curl 'http://127.0.0.1:8080/indexes/myindex/search?q=botman'
 
 ## License 
 
-Common clause
\ No newline at end of file
+Common clause
```

Comment:
What is that ? Is it ascii italic? did you put `_` around the text?

Reply:
It has been changed to more specific licence without italic